### ISSUE TITLE
rust-clippy website add new lint need displayed the current version.

### ISSUE THREAD

**Issue by baoyachi** [rust-lang/rust-clippy#8641]
### Description

When I use `rust nightly `channel ,with `cargo-clippy` also updated.But  I don't know in which version this clippy lint error was added.

When I goto detail error with `clippy` website,but there no version with `cargo-clippy`. 

### Version

```text
rustc 1.59.0 (9d1b2106e 2022-02-23)
binary: rustc
commit-hash: 9d1b2106e23b1abd32fce1f17267604a5102f57a
commit-date: 2022-02-23
host: x86_64-apple-darwin
release: 1.59.0
LLVM version: 13.0.0
```

### Additional Labels

@rustbot label +C-enhancement

**Comment by xFrednet** [comment]
The lint description contains a field that lists the rust version that the lint was added. Clippy's version follows the versioning of rust. Here is an example how we currently display the version:

![IMG_20220406_151902_106.jpg](https://user-images.githubusercontent.com/17087237/161984326-7f5f959a-f300-4aa8-9fc2-846b0c5c76b3.jpg)

I think it works better in the description itself, since it's less important for most people. was this what you were suggesting? 🙃

**Comment by baoyachi** [comment]
> The lint description contains a field that lists the rust version that the lint was added. Clippy's version follows the versioning of rust. Here is an example how we currently display the version:
> 
> ![IMG_20220406_151902_106.jpg](https://user-images.githubusercontent.com/17087237/161984326-7f5f959a-f300-4aa8-9fc2-846b0c5c76b3.jpg)
> 
> I think it works better in the description itself, since it's less important for most people. was this what you were suggesting? 🙃

It's rust version,not `rust-clippy` version, beacase when I use rust nighlty channel, `rust-clippy` also be update.

**Comment by xFrednet** [comment]
`rust-clippy`'s version follows the version of rustc. The example lint was released with Clippy `0.1.52`. We've decided to list the rust version as "Added in `0.1.52`" could be confusing to everyone who doesn't know the versioning of Clippy :upside_down_face:
